notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>I think it make sense to demonstrate which inputs used for running pipeline.</p><p>Could you please show pandas data frame?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done